### PR TITLE
Added homework_9_071222 (Tasks 1-3)

### PR DESCRIPTION
**Pull request by kate-lvlup**
Hi Dima,
Could you check my homework9 today?
Thanks in advance and have a great Friday evening :)

### CODE REVIEW

File: homework_9_071222/src/myMathMethods/MyMath.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        if (a >= b) {
+            return b;
+        } else {
+            return a;
+        }
+    }
+
+    //    находнения большего целого числа, меньшего целого числа - округление(на вход число с плавающей точкой)
+    public static double maxValue(double x, double y) {
+        x = Math.round(x);
```
Comment (YeaaaaH) [inline homework_9_071222/src/myMathMethods/MyMath.java:62]: what if you get x = 2.6, y=2.5?

File: homework_9_071222/src/myMathMethods/MyMath.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        y = Math.round(y);
+        if (x >= y) {
+            return x;
+        } else {
+            return y;
+        }
+    }
+
+    public static double minValue(double x, double y) {
+        x = Math.round(x);
```
Comment (YeaaaaH) [inline homework_9_071222/src/myMathMethods/MyMath.java:72]: same as previous

File: homework_9_071222/src/strings/TakeString.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+package strings;
+
+public class TakeString {
+
+    public static int numberVowels(String string) {
+        int number = 0;
+        char[] charArray = string.toCharArray();
+        for (int i = 0; i < charArray.length; i++) {
+            if (charArray[i] == 65 || charArray[i] == 69 || charArray[i] == 73 || charArray[i] == 79 || charArray[i] == 85
```
Comment (YeaaaaH) [inline homework_9_071222/src/strings/TakeString.java:14]: you don't need uppercase letters

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
small fixes needed

File: homework_9_071222/src/myMathMethods/MyMath.java (hunk 1 above)
Comment (kate-lvlup) [reply homework_9_071222/src/myMathMethods/MyMath.java:62]: Fixed. Numbers are compared first, then rounded up.

File: homework_9_071222/src/myMathMethods/MyMath.java (hunk 2 above)
Comment (kate-lvlup) [reply homework_9_071222/src/myMathMethods/MyMath.java:72]: Fixed. Numbers are compared first, then rounded up.

File: homework_9_071222/src/strings/TakeString.java (hunk 3 above)
Comment (kate-lvlup) [reply homework_9_071222/src/strings/TakeString.java:14]: Fixed.

**Review by YeaaaaH — APPROVED** [review]

**Pull request merged**